pkg/controller/lifecycle/actuator.go

```diff
@@ -525,14 +512,6 @@ func getSeedResources(oidcReplicas *int32, hibernated bool, namespace, genericKu
 		}
 	}
 
-	if oidcReplicas != nil && *oidcReplicas > 0 {
-		err = registry.Add(buildHPA(namespace))
```

Comment:
I think we should bump the default replicas to 2, which is currently done behind the scenes by a webhook, but to be on the safe side it will be nice this controller also to set the replicas to something `> 1`.

Reply:
Done with https://github.com/gardener/gardener-extension-shoot-oidc-service/pull/243/commits/23d0f036a69956e3ee45a69b947487b1e320f2fd PTAL